settings.gradle

```diff
@@ -30,7 +30,7 @@ include ":${projectName}-examples"
 project(':checkstyle').projectDir = file("build-config${File.separator}checkstyle")
 project(":${projectName}-native").projectDir = file('native')
 project(":${projectName}-ballerina").projectDir = file('ballerina')
-project(":${projectName}-examples").projectDir = file('examples')
+project(":${projectName}-examples").projectDir = file('samples')
```

Comment:
our connector convention is to use a single directory called `examples` to house all the samples related to the particular module. Is there any reason to deviate from that?

Reply:
This was corrected in [db871a1](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/322/commits/db871a17080bb612b6313fc1db13a6ae72c0d39f).